transcom/mymove
# [MB-15811] Adding test harness data

## [Jira ticket](https://dp3.atlassian.net/browse/MB-15811)

## Summary

This PR adds a new factory for `SITAddressUpdates`, a subscenario for devseed data (`subScenarioSITAddressUpdates`), and playwright test harness data for incoming SIT work

We're not sure if `HHGMoveInSITWithAddressChangeRequestUnder50Miles` will be needed for playwright tests since those shouldn't require action by the TOO, but it can be removed later if it ends up being unused.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. `go test -count 1 -v ./pkg/factory` to make sure factory tests pass
2. `DEVSEED_SUBSCENARIO=sit_address_updates make db_dev_e2e_populate`
3. Log in as a TOO and check that the moves are visible 
4.  Check that the `sit_address_updates` table in your db has some data in it
